src/pages/Question.tsx

```diff
@@ -40,79 +38,81 @@ const MOVIES = gql`
   }
 `
 
+const GET_SCORE = gql`
+  query($imdbId: String!, $selectedTitle: String!, $remainingSeconds: Int!) {
+    scoreTitleResponse(
+      imdbId: $imdbId
+      selectedTitle: $selectedTitle
+      remainingSeconds: $remainingSeconds
+    )
+  }
+`
+
 const Question = () => {
   const history = useHistory()
 
-  useStorage()
-
-  const { addPoints, removeLife, currentImdbId, timeRemaining } = useStore()
+  const {
+    addPoints,
+    removeLife,
+    currentImdbId,
+    timeRemaining,
+    setTimeRemaining,
+  } = useStore()
 
   const { loading, error, data } = useQuery<MovieData>(MOVIES, {
-    variables: { imdbId: currentImdbId },
+    fetchPolicy: 'cache-only',
   })
 
-  const [movies, setMovies] = useState<Movie[]>([])
-
-  useIonViewWillEnter(() => {
-    if (data) {
-      const movie: Movie = {
-        imdbId: data.movie.imdbId,
-        title: data.movie.title,
-        releaseYear: data.movie.releaseYear,
-        posterPath: data.movie.posterPath,
-        randomMovies: [],
-      }
-
-      const randomMovies: Movie[] = [...data.movie.randomMovies]
-
-      // combine all movie object into one array
-      const _movies = [movie, ...randomMovies]
-
-      setMovies(randomArrayShuffle(_movies))
-    }
-  }, [data])
+  const [submitResponse, scoreResponse] = useLazyQuery(GET_SCORE)
```

Comment:
We use a lazy query here so that we can trigger it in out button onClick and we can then use an effect to do an evaluation on changes of scoreResponse. That is a pattern I use quite often.